docker/dev/Dockerfile

```diff
@@ -58,6 +58,13 @@ RUN apt-get update && \
 RUN groupadd -g 1000 ubuntu && \
     useradd -u 1000 -g ubuntu -m -N -s /bin/bash ubuntu
 
+# We are adding sudo capabilities to the ubuntu user so that devs can run the cmake build inside the container.
+# If we instead have devs use the root user when they execute parflow you get MPI errors
+RUN apt-get update \
+    && apt-get install -y sudo
+RUN adduser ubuntu sudo
+RUN echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
```

Comment:
I know it is a weird thing to do to add sudo to a docker container but I think we need to because
1) You cannot run parflow as root inside the docker container or MPI will throw errors
2) You need root privileges to rebuild the code base within the container
So if we want to support people being able to iteratively rebuild within the container and also using it to execute parflow on test cases, they need to be logged in as ubunutu but also have root privelages